src/main/java/io/confluent/castle/cluster/CastleClusterConf.java

```diff
@@ -52,11 +55,17 @@ public void validateKafkaPath() {
         }
     }
 
-    public void validateCastlePath() {
-        if (castlePath.isEmpty() ||
-                !(new File(castlePath).isDirectory())) {
-            throw new RuntimeException("The current value of castlePath (" +
-                castlePath + ") does not point to a valid directory.");
+    @JsonProperty
+    public String schemaRegistryPath() {
+        return schemaRegistryPath;
+    }
+
+    public void validateSchemaRegistryPath() {
+        // Schema registry path is optional
+        if (!schemaRegistryPath.isEmpty() &&
+                !(new File(schemaRegistryPath).isDirectory())) {
```

Comment:
nit: `Files.isDirectory` might be slightly nicer

Reply:
This is copied from `validateKafkaPath`.  I tried changing it, it would be `Files.isDirectory(Paths.get(schemaRegistryPath)))`, which is about the same, so I think I'll leave it :)

Reply:
ok